README.md

```diff
@@ -19,17 +19,48 @@ $> npm install react-onclickoutside --save
 var onClickOutside = require('react-onclickoutside');
 
 // create a new component, wrapped by this onclickoutside HOC:
-var MyComponent = onClickOutside(React.createClass({
+var MyComponent = onClickOutside(
```

Comment:
I had it similarly to what you are proposing in a previous version, but I decided to go with the idiom used in other HOC's such as [connect in react-redux](https://github.com/reactjs/react-redux/blob/master/docs/api.md#connectmapstatetoprops-mapdispatchtoprops-mergeprops-options).

If you really want, I can change it.